config/default-lsst.psfex

```diff
@@ -11,8 +11,8 @@ PSF_ACCURACY    0.01            # Accuracy to expect from PSF "pixel" values
 #------------------------- Point source measurements -------------------------
  
 CENTER_KEYS     slot_Centroid_x,slot_Centroid_y # Catalogue parameters for source pre-centering
-PHOTFLUX_KEY    slot_ApFlux_flux    # Catalogue parameter for photometric norm.
-PHOTFLUXERR_KEY slot_ApFlux_fluxErr # Catalogue parameter for photometric error
+PHOTFLUX_KEY    base_CircularApertureFlux_9_0_flux      # Catalogue parameter for photometric norm
+PHOTFLUXERR_KEY base_CircularApertureFlux_9_0_fluxSigma # Catalogue parameter for photometric error
```

Comment:
This raises the whole question of how magic numbers are tied together (e.g. semantic labels on pex_config properties).  Out of scope for this issue.